appveyor.yml

```diff
@@ -1,10 +1,19 @@
-version: 4.0.0.{build}
+version: 4.1.0.{build}
 platform:
-    - x64
+  - x64
```

Comment:
I'm not sure that using platform here gives us any value. Where did you find this?